hpy/devel/src/runtime/ctx_call.c

```diff
@@ -29,8 +29,8 @@ ctx_CallTupleDict(HPyContext *ctx, HPy callable, HPy args, HPy kw)
     else {
         // args is null, but kw is not, so we need to create an empty args tuple
         // for CPython's PyObject_Call
-        HPy items[] = {};
-        HPy empty_tuple = HPyTuple_FromArray(ctx, items, 0);
+        HPyTupleBuilder builder = HPyTupleBuilder_New(ctx, 0);
+        HPy empty_tuple = HPyTupleBuilder_Build(ctx, builder);
         obj = PyObject_Call(_h2py(callable), _h2py(empty_tuple), _h2py(kw));
```

Comment:
what about using `HPy *items = NULL` instead? It should work fine with `HPyTuple_FromArray`, and `HPyTupleBuilder` looks overkill for this simple task

Reply:
Seems to work fine! I changed my PR

Reply:
thank you!